client/src/components/arabicListWords/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import './style.css'
+
+class arabicListWords extends Component {
+  state = {
+    arabicList: []
+  }
+
+  componentDidMount() {
+    axios.get(`api/arabic-words`).then(({ data }) => {
+      this.setState({ arabicList: data })
+    })
+  }
+
+  render() {
+    return (
+      <div>
+        <h1> List Of Arabic Words </h1>
+        <React.Fragment>
+          {this.state.arabicList.map(element => (
```

Comment:
You can destructure element here also in the parameter as you're repeating this below.